### ISSUE TITLE
Download feature is not working at all

### ISSUE THREAD

**Issue by matsumurae** [AcrylicShrimp/dlsite-manager#14]
Hello~

Seems like there was some update on DL as it doesn't download any manga (I was already aware it doesn't work with audio).

Steps:
1. Login into my account
2. Shows my products list
3. Click on download. I choose a manga that doesn't allow to download, only browser.
4. DLSite folder appears empty but it the app shows "download" button available again (so it did "something").

**Comment by AcrylicShrimp** [comment]
Thanks for reporting! I tested and it's reproduced. Let me work on it, it will be fixed in next major release.

**Comment by AcrylicShrimp** [comment]
Fixed in the latest release.

**Comment by matsumurae** [comment]
> Fixed in the latest release.

Thanks! I was excited to try but seems like there's no mac release. I can't find any .dmg file on the latest release, will be available?

**Comment by AcrylicShrimp** [comment]
@matsumurae Ah, yes. The macOS builds will be available in morning.

**Comment by AcrylicShrimp** [comment]
@matsumurae macOS binary is added.

**Comment by matsumurae** [comment]
> @matsumurae macOS binary is added.

Thanks! Just checked and seems like it doesn't work. Tested with Manga and Voice (just to check, I already downloaded them). Manga code is BJ135282, Voice is BJ01349921.

I've tried both compress and w/o compress in Manga and both downloaded the same, folder with ID > inside was a 14kb .zip. For sure it's wrong, it's impossible to just weight 14kb.

I would be happy to have an option to download manga, as that's the impossible to trick on website.
